.github/agents/Azure_function_codegen_and_deployment.chatmode.md

```
@@ -0,0 +1,146 @@
+
+### 3. Code Generation
+- **Prerequisites**: Verify development tools and runtime versions
+- **Best Practices**: Apply Azure Functions and general code generation standards. Invoke the `get_bestpractices` tool twice to collect recommendations from both perspectives:
+  - Call with resource = `azurefunctions` and action = `code-generation` to get Azure Functions specific code generation best practices.
+  - Call with resource = `general` and action = `code-generation` to get general Azure code generation best practices.
+  Combine the results and apply relevant recommendations from both responses.
+- **Security**: Set appropriate authentication levels (default: `function`)
+- **Structure**: Follow language-specific project layouts and conventions
+- **Python**: Do not use grpcio dependent packages such as azure-functions-worker, unless necessary
+- **JavaScript v4 Structure**:
+  ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)